ddl/lightning/env.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
Could we add some UT for this file?

Reply:
Updated. BTW `genRLimit` is a wrapper for system call, it is meaningless to test it..